api/controllers/apps_test.go

```diff
@@ -55,8 +60,11 @@ func TestAppShowWithAppNotFound(t *testing.T) {
 
 func TestAppCreate(t *testing.T) {
 	aws := test.StubAws(
-		test.CreateAppStackCycle("application"),
-		test.DescribeAppStackCycle("application"),
+		test.DescribeStackNotFound("application"),
+		test.DescribeStackNotFound("convox-test-application"),
+		test.CreateAppStackCycle("convox-test-application"),
+		test.DescribeStackNotFound("application"),
+		test.DescribeAppStackCycle("convox-test-application"),
```

Comment:
Related to the comment in AppCreate, I don't see why POST /apps needs to make more CF calls. It should be able to stick with 2 API calls: CreateAppStack for convox-test-application, and DescribeAppStack for convox-test-application.